packages/core/src/transports/http.ts

```diff
@@ -94,7 +95,7 @@ export function http<
     batch,
     fetchOptions,
     key = 'http',
-    methods,
+    methods = { exclude: UTXOAPISchemaMethods as unknown as string[] },
```

Comment:
@chybisov  Is there a way we can type this without going through `unknown`?

Reply:
In this case, UTXOAPISchemaMethods won't be `readonly`, but I changed it :)